runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -515,7 +515,8 @@ private void checkClusterId(@Nonnull Layout layout) {
                     UuidUtils.asBase64(clusterId));
         } else if (!clusterId.equals(layout.getClusterId())) {
             // We connected but got a cluster id we didn't expect.
-            throw new WrongClusterException(clusterId, layout.getClusterId());
+            //throw new WrongClusterException(clusterId, layout.getClusterId());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)